doc/contribute/external.rst

```diff
@@ -103,6 +103,32 @@ With this approach the code is considered as being developed externally, and
 thus it is not automatically subject to the requirements of the previous
 section.
 
+Integration in main manifest file (west.yaml)
+++++++++++++++++++++++++++++++++++++++++++++++
+
+Integrating external code into the main :file:`west.yml` manifest file is
+limited to code that is used by a Zephyr subsystem (libraries) or by a platform
+or drivers (HAL) or tooling needed to test or build Zephyr components.
```

Comment:
How about adding some language that indicates the testing/support of integrated modules?  Something like "Modules in this group are validated by the CI, and verified working with each Zephyr release."